src/chapters/loops/choosing-loops.rst

```diff
@@ -0,0 +1,22 @@
+Choosing Which Loop to Use
+==========================
+
+.. index::
+   single: iteration; definite
+   single: iteration; indefinite
+
+The ``for`` loop is typically used to iterate through a fixed set of values that can be determined before the loop executes. This is why it is often said that a ``for`` loop exhibits **definite iteration**.
```

Comment:
Word reduction: “This is why we say a ``for`` loop exhibits **definite iteration**.”